resources/views/livewire/airfields/index.blade.php

```diff
@@ -0,0 +1,21 @@
+<div class="mb-4">
+    <header class="flex flex-col mt-4">
+        <div class="flex flex-col text-center md:text-left md:flex-row justify-between">
+            <h1 class="text-white text-5xl font-medium ">Airfields</h1>
+            <input type="text" wire:model="search" placeholder="Search by ICAO..." class="rounded mt-2 md:mt-0">
```

Comment:
`wire:model` should use `lazy` or `debounce` to prevent spamming the API. Right now every character being types into this search is triggering an API call.

Ref: https://laravel-livewire.com/docs/2.x/reference

Reply:
done